.codesandbox/ci.json

```diff
@@ -1,12 +1,9 @@
 {
   "packages": [
-    "packages/babel-plugin-emotion",
+    "packages/babel-plugin",
     "packages/cache",
-    "packages/core",
-    "packages/create-emotion",
+    "packages/react",
```

Comment:
we might want to add `sheet` here as well, or maybe just use a wildcard for simplicity